cnxepub/models.py

```diff
@@ -185,7 +196,7 @@ def flatten_to(model, flatten_filter):
     for m in flatten_model(model):
         if flatten_filter(m):
             yield m
-    raise StopIteration()
+    return
```

Comment:
Python 3 complained about raise StopIteration() as deprecated, said an explicit return does that. I just tested, and in fact, implicit does as well. We can drop them completely.